citation-react-router/src/Routes.js

```diff
@@ -33,7 +33,8 @@ export default class Routes extends Component {
 		}
 	}
 
-	createElement(page, { type, data, props, children = [] }, i, matchProps) {
+	createElement(page, { type, data, props, children }, i, matchProps) {
+		children = children ? children : [];
 		let Component = this.props.components[type];
```

Comment:
With the modification which delete empty files, null value can be passed here but null value doesn't trigger the default value